mysql-async/src/main/java/com/github/jasync/sql/db/mysql/codec/MySQLConnectionHandler.kt

```diff
@@ -4,54 +4,19 @@ import com.github.jasync.sql.db.Configuration
 import com.github.jasync.sql.db.exceptions.DatabaseException
 import com.github.jasync.sql.db.general.MutableResultSet
 import com.github.jasync.sql.db.mysql.binary.BinaryRowDecoder
-import com.github.jasync.sql.db.mysql.message.client.AuthenticationSwitchResponse
-import com.github.jasync.sql.db.mysql.message.client.CapabilityRequestMessage
-import com.github.jasync.sql.db.mysql.message.client.CloseStatementMessage
-import com.github.jasync.sql.db.mysql.message.client.HandshakeResponseMessage
-import com.github.jasync.sql.db.mysql.message.client.PreparedStatementExecuteMessage
-import com.github.jasync.sql.db.mysql.message.client.PreparedStatementPrepareMessage
-import com.github.jasync.sql.db.mysql.message.client.QueryMessage
-import com.github.jasync.sql.db.mysql.message.client.QuitMessage
-import com.github.jasync.sql.db.mysql.message.client.SendLongDataMessage
-import com.github.jasync.sql.db.mysql.message.server.AuthenticationSwitchRequest
-import com.github.jasync.sql.db.mysql.message.server.BinaryRowMessage
-import com.github.jasync.sql.db.mysql.message.server.ColumnDefinitionMessage
-import com.github.jasync.sql.db.mysql.message.server.EOFMessage
-import com.github.jasync.sql.db.mysql.message.server.ErrorMessage
-import com.github.jasync.sql.db.mysql.message.server.HandshakeMessage
-import com.github.jasync.sql.db.mysql.message.server.OkMessage
-import com.github.jasync.sql.db.mysql.message.server.PreparedStatementPrepareResponse
-import com.github.jasync.sql.db.mysql.message.server.ResultSetRowMessage
-import com.github.jasync.sql.db.mysql.message.server.ServerMessage
+import com.github.jasync.sql.db.mysql.message.client.*
```

Comment:
please refrain from using star imports

Reply:
My IDEA with default JetBrains code style automatic format to *, could you please provide your code style xml so i can import to my IDEA.

Reply:
see https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports/25916603

Reply:
Thanks